### DISCUSSION TITLE
How to check if a dropdown is open?

### DISCUSSION THREAD

**Discussion by jmrossy** [saadeghi/daisyui#2697] [Q&A] [answered]
How can I use Javascript to check if a dropdown's content is currently visible?

An example scenario is that I have hooks in react-based dropdown content that I want to run only when the dropdown is open.

**Comment by saadeghi** [comment, marked as answer]
[Method 1 dropdown](https://daisyui.com/components/dropdown/#method-1-using-details-and-summary-tags) works with HTML `<details>` tag so you can use the `ontoggle` event to check if it's open or not:  
https://www.cssportal.com/html-event-attributes/ontoggle.php

[Method 2 dropdown](https://daisyui.com/components/dropdown/#method-2-using-css-focus) works with CSS focus so you can use `onfocus` event:  
https://www.cssportal.com/html-event-attributes/onfocus.php

Let me know if you have a question.

**Reply by jmrossy** [reply]
Perfect, thank you!
